NavigationReactNative/src/TabBar.tsx

```diff
@@ -54,9 +54,20 @@ class TabBar extends React.Component<any, any> {
         var TabBar = primary ? NVTabBar : TabBarPager;
         var TabView = primary ? NVTabNavigation : (!I18nManager.isRTL ? NVTabLayout : NVTabLayoutRTL);
         TabView = Platform.OS === 'android' ? TabView : NVSegmentedTab;
+        // Default to auto
+        var labelVisibilityVal = -1;
+
+        if(labelVisibility === "selected") {
+          labelVisibilityVal = 0;
+        } else if (labelVisibility === "show") {
+          labelVisibilityVal = 1;
+        } else if (labelVisibility === "hide") {
+          labelVisibilityVal = 2;
+        }
```

Comment:
Instead of hard-coding these constants, you can get them from the java code like [I've done in the SearchBar](https://github.com/grahammendick/navigation/blob/9e9c0f7efc3906581f8b8a0720ff35f75305142f/NavigationReactNative/src/SearchBar.tsx#L36). You need to [export them from the view manager](https://github.com/grahammendick/navigation/blob/9e9c0f7efc3906581f8b8a0720ff35f75305142f/NavigationReactNative/src/android/src/main/java/com/navigation/reactnative/SearchBarManager.java#L80)

Reply:
I've had a free hour today so I've been working on this, but I cannot figure out how to do as you suggest. I can't seem get the constants from any Android classes, the docs say they exist on NavigationBarView but we're overriding that with the React Native implementation which does not export those constants. Any advice on how to do this?

Reply:
Thanks for picking this up again. You probably need to fully qualify the constants with the namespace so picks the right class. For example, 
```java
com.google.android.material.navigation.NavigationBarView.LABEL_VISIBILITY_SELECTED
```

Reply:
Aha, didn't know I could do it like that; I'd tried adding it as an import at the top in an attempt to get it using the right one but this is much more direct! Thanks